spec/web_helpers.rb

```diff
@@ -1,7 +1,8 @@
-def create_space
+def create_space(name, description="")
```

Comment:
We did this as we didn't make description as a required field. Do you think it needs to be?